### PR TITLE
model.yaml draft [WIP, RFC]

### PR DESCRIPTION
**Pull request by FynnBe**
Configuring @wolny's 3dUnet for the model zoo...

### CODE REVIEW

File: pytorch3dunet/bioimage-io/UNet3DArabidopsisOvules.model.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  - Adrian Wolny;@bioimage-io
+documentation: ../README.md
+tags: [unet3d, pytorch, arabidopsis, ovuls, cell membrane, segmentation, plant tissue]
+
+format_version: 0.1.0
+language: python
+framework: pytorch
+
+# todo: import path with '..'  # add a root_dir?
+source: ..pytorch3dunet.model.UNet3D
```
Comment (FynnBe) [inline pytorch3dunet/bioimage-io/UNet3DArabidopsisOvules.model.yaml]: relative import paths with `..` should probably be avoided...
- Do we need a root_dir? with default `root_dir = "."`? < current favorite
- Is the repo dir `root_dir=repo_dir`? Do we then check  from "." and repo_dir?

File: pytorch3dunet/bioimage-io/UNet3DArabidopsisOvules.model.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+name: UNet3DArabidopsisOvules
+description: A 3d U-Net trained to predict the cell boundaries in confocal stacks of Arabidopsis ovules
+cite:
+  - text: "Wolny, Adrian et al. Accurate and Versatile 3D Segmentation of Plant Tissues at Cellular Resolution. BioRxiv 2020."
+    doi: https://doi.org/10.1101/2020.01.17.910562
+authors:
+  - Adrian Wolny;@bioimage-io
```
Comment (FynnBe) [inline pytorch3dunet/bioimage-io/UNet3DArabidopsisOvules.model.yaml]: We should refine the author string format, do we link user names or orgas? other tags?

File: bioimage-io/UNet3DArabidopsisOvules.model.yaml [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+test_output: null # ../test_output.npy
+thumbnail: null # ./nuclei_thumbnail.png
+
+# TODO inputs/outputs
+#inputs:
+#  - name: raw
+#    axes: bcyx
+#    data_type: float32
+#    data_range: [-inf, inf]
+#    shape: [1, 1, 128, 128]
```
Comment (wolny) [inline bioimage-io/UNet3DArabidopsisOvules.model.yaml]: what is the exact semantics of the `shape` param in here?

File: bioimage-io/UNet3DArabidopsisOvules.model.yaml [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+##        min: [1, 1, 32, 32]
+##        step: [0, 0, 32, 32]
+#outputs:
+#  - name: output
+#    axes: bcyx
+#    data_type: float32
+#    data_range: [0, 1]
+#    shape: [1, 1, 128, 128]
+#    halo: [0, 0, 32, 32]
+##    shape:
```
Comment (wolny) [inline bioimage-io/UNet3DArabidopsisOvules.model.yaml]: `shape` key is duplicated in under the `outputs`

**Review by wolny — APPROVED** [review]
LGTM :+1:

**Pull request merged**